src/App.js

```diff
@@ -1,16 +1,47 @@
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import { Login } from "./Components/Auth/Login/login";
 import { Navbar } from "./Components/Navbar/navbar";
+import { PlaylistModal } from "./Components/PlaylistModal/playlistmodal";
+import { PrivateRoute } from "./Components/PrivateRoute/privateRoute";
+import { HistoryVideo } from "./Pages/History/history";
+import { LikedVideo } from "./Pages/Liked-Videos/likeVideo";
 import { SingleVideo } from "./Pages/Single-Video-Page/single-video";
 import { VideoListing } from "./Pages/Video-Listing/videoListing";
+import { WatchLater } from "./Pages/WatchLater/watchLater";
```

Comment:
suggestion: you can export all the pages from one file and import all for a better code look.😊